src/transformations/transformations.ts

```diff
@@ -291,6 +296,9 @@ export const isTextTransformation = (x: Transformation): x is TextTransformation
     x.type === 'moveCursor'
   );
 };
+export const isMultiCursorTextTransformation = (x: Transformation): Boolean => {
+  return (x.type === 'insertTextVSCode' && x.isMultiCursor) ?? false;
+};
 
 const getRangeFromTextTransformation = (transformation: TextTransformations): Range | undefined => {
```

Comment:
Is there a reason these aren't defined with `function`?

Reply:
I was just trying to keep style consistency of the document and didn't think it'd make a difference. Should I change them?

Reply:
Up to you, just making sure I wasn't missing something